packages/mdc-list/component.ts

```diff
@@ -129,6 +129,10 @@ export class MDCList extends MDCComponent<MDCListFoundation> {
     }
   }
 
+  setEnabled(itemIndex: number, isEnabled: boolean) {
+    this.foundation_.setEnabled(itemIndex, isEnabled);
```

Comment:
You also need tests for this. :) You wouldn't need to mock since you'll be testing with real DOM for component.

Reply:
sounds good! I followed the same testing format as the initializeListType method in the component to continue the pattern but if let me know if you think there is a better way to test it?